src/app/modules/myinfo/myinfo.util.ts

```diff
@@ -109,6 +166,12 @@ export const compareHashedValues = (
   const myInfoResponsesMap: MyInfoComparePromises = new Map()
   responses.forEach((field) => {
     if (hasMyInfoAnswer(field)) {
+      // Children birth records have multiple possible hash values so they
+      // need to be checked one by one.
+      if (field.myInfo.attr === MyInfoAttribute.ChildrenBirthRecords) {
+        handleMyInfoChildHashResponse(field, hashes, myInfoResponsesMap)
```

Comment:
Same comment on mutating the Map, but I realize that in this case, there's no equivalent to `assign()` or `merge()` for Maps 😑 , so adding code here to do the merge is sort of polluting. 

I think you can ignore this comment and the previous one and leave as-is.